src/window.rs

```diff
@@ -1396,6 +1396,19 @@ impl Window {
             .maybe_wait_on_main(|w| w.drag_resize_window(direction))
     }
 
+    /// Show window menu at current mouse position.
+    ///
+    /// There's no guarantee that calling this method show the window menu.
+    /// 
+    /// ## Platform-specific
+    /// - **Windows:** Window need to have `WS_SYSMENU` window style to be able to show the [`window menu`].
+    /// - **Android / iOS / macOS / Orbital / Wayland / Web / X11:** Unsupported.
+    /// 
+    /// [`window menu`]: https://en.wikipedia.org/wiki/Common_menus_in_Microsoft_Windows#System_menu
```

Comment:
```suggestion
    /// 
    /// [`window menu`]: https://en.wikipedia.org/wiki/Common_menus_in_Microsoft_Windows#System_menu
```
I guess we can skip the link, I can't really come up with wording that makes it useful to incorporate it.

Reply:
There is currently nothing linking to it, so if you want to keep it you have to use it somewhere.